### PR TITLE
Add ID prop for Autocomplete component

### PR DESCRIPTION
**Pull request by mlileikis**
For such a small change I did not create new branch or tests, please review.

### CODE REVIEW

**Comment by alextrastero** [comment]
Maybe you could also write a spec? That would be great! :)

**Comment by mlileikis** [comment]
Pushed to my fork, do I need to update this PR/ create a new one?
/**
 * BTW, these are javadocs in Java. I noticed you document like that too, what are these called like in JS?
 */

File: src/Autocomplete.js [hunk 1]
```diff
@@ -101,6 +101,7 @@ class Autocomplete extends Component {
       ...props
     } = this.props;
 
+	const _id = {id || `autocomplete-${idgen()}`};
```
Comment (alextrastero) [inline src/Autocomplete.js]: The brackets are causing the build to fail :), no need for them

**Comment by alextrastero** [comment]
You could remove https://github.com/react-materialize/react-materialize/blob/master/test/DropdownButtonSpec.js#L40 spec, since it's already covered somewhere else

**Comment by mlileikis** [comment]
I added test for ID prop and updated failing test. Now locally I get 'npm test' passing

**Comment by alextrastero** [comment]
Perfect! thanks man, please continue contributing! :1st_place_medal:

**Comment by alextrastero** [comment]
@mlileikis did you close this by mistake?

**Comment by mlileikis** [comment]
I thought I should close it manually. Eh my bad, I'll leave it opened

**Comment by alextrastero** [comment]
:) i'll create a new build with this PR and another one ASAP

**Comment by alextrastero** [comment]
Released on `1.1.1`

**Pull request merged**